metrics/metrics.go

```diff
@@ -88,8 +88,8 @@ func (m *performanceMetric) Failure() {
 type NullablePerformanceMetric struct{}
 
 func (NullablePerformanceMetric) Start() time.Time {
-	return time.Now()
+	return time.Time{}
```

Comment:
Could you please add a comment here for better understanding?

Reply:
added; let me know if it is good :)